ratelimit.go

```diff
@@ -0,0 +1,87 @@
+package strava
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type RateLimit struct {
+	NextRequestTime time.Time
```

Comment:
I really feel the `NextRequestTime` stuff is specific to your use case. I think the library should just report the info and let the user figure out how to interpret it.